renovate.json

```diff
@@ -3,5 +3,6 @@
   "extends": ["config:base"],
   "labels": ["dependency"],
   "prConcurrentLimit": 5,
-  "rangeStrategy": "pin"
+  "rangeStrategy": "pin",
+  "schedule": ["monthly"]
```

Comment:
Are you sure this is a valid schedule value? I think schedule:monthly is actually a preset, set above in "extends". I could be wrong though.

I've previously used https://github.com/marketplace/actions/validate-renovate-configuration-with-renovate-config-validator which would confirm this for us. Worth adding?

Reply:
From my understanding, yeah, it is valid. The CLI also approved it, just applied one suggestion! Will add that to the validations as well I suppose!

Reply:
Nice, good stuff. Thanks for adding and checking!